packages/core/src/shaderlib/extra/unlit.fs.glsl

```diff
@@ -8,11 +8,23 @@ uniform float u_alphaCutoff;
     uniform sampler2D u_baseTexture;
 #endif
 
+vec4 gammaToLinear(vec4 srgbIn){
```

Comment:
Write a public `Oasis.glsl`, put the basic  fun like `gammaToLinear()` 、`linearToGamma()` and 、decodeHDRTexture() in it.